### ISSUE TITLE
`-ms-interpolation-mode` property warned as invalid

### ISSUE THREAD

**Issue by awayken** [rossPatton/stylint#62]
The `-ms-interpolation-mode` property is warned as being invalid. Interpolation mode is mostly used in my code to help improve IE 6 and 7 rendering of images that are scaled down. CSS-Tricks mentions it in their article [IE Fix: Bicubic Scaling for Images](https://css-tricks.com/ie-fix-bicubic-scaling-for-images/).

bicubic.styl
```stylus
img {
    -ms-interpolation-mode: bicubic;
}
```

bicubic.json
```json
{
    "alphabetical": true,
    "borderNone": true,
    "brackets": false,
    "colons": false,
    "colors": false,
    "commaSpace": true,
    "commentSpace": false,
    "cssLiteral": false,
    "depthLimit": false,
    "duplicates": true,
    "efficient": true,
    "enforceBlockStyle": false,
    "enforceVarStyle": false,
    "extendPref": false,
    "globalDupe": false,
    "indentSpaces": 4,
    "leadingZero": true,
    "maxWarnings": 10,
    "maxWarningsKill": false,
    "mixed": false,
    "namingConvention": false,
    "parenSpace": false,
    "placeholders": true,
    "semicolons": false,
    "universal": true,
    "valid": true,
    "whitespace": true,
    "zeroUnits": true,
    "zIndexDuplicates": false,
    "zIndexNormalize": false
}
```

Output:
```bash
Warning:  property is not valid
File: bicubic.styl
Line: 2: -ms-interpolation-mode: bicubic;
```

**Comment by rossPatton** [comment]
this will be fixed in the next update

**Comment by awayken** [comment]
Tested this on develop, and it looks good!
